modules/floating-buttons/assets/scss/widgets/contact-buttons.scss

```diff
@@ -57,7 +57,7 @@
 	gap: 20px;
 	position: fixed;
 	width: var(--e-contact-buttons-chat-box-width);
-	z-index: 99;
+	z-index: 10000;
```

Comment:
@gabriel-glo there are z-index variables in the `variables.scss` file in the elementor code. Can you please use one of those?

Reply:
@mserino Done.